### PR TITLE
boards: stm32l496g_disco: Minor fixes on dts and docs

### PR DESCRIPTION
**Pull request by gon1332**
This PR addresses the following issues:

- LPUART 1 mapping was not correct
- The sublists on the docs did not appear as expected

### CODE REVIEW

File: boards/arm/stm32l496g_disco/doc/stm32l496g_disco.rst [hunk 1]
```diff
@@ -117,8 +117,8 @@ The STM32L496AG SoC provides the following hardware capabilities:
 
 
 More information about STM32L496AG can be found here:
-       - `STM32L496AG on www.st.com`_
-       - `STM32L496 reference manual`_
+  - `STM32L496AG on www.st.com`_
```
Comment (dbkinder) [inline boards/arm/stm32l496g_disco/doc/stm32l496g_disco.rst]: Better would be:
```
More information about STM32L496AG can be found in:

- `STM32L496AG on www.st.com`_
- `STM32L496 reference manual`_

```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Thanks for fixing all those extra indents... one more tweak

**Review by dbkinder — APPROVED** [review]
Nice, thanks.

**Review by erwango — APPROVED** [review]

**Comment by erwango** [comment]
recheck

**Comment by gon1332** [comment]
@erwango It seems that kernel timer api fails one check. It's strange, since my change seems trivial.

**Comment by erwango** [comment]
recheck

**Comment by erwango** [comment]
@gon1332 

>@erwango It seems that kernel timer api fails one check. It's strange, since my change seems trivial.

Yeah, I don't know exactly why you'd be the only one impacted.
Can you rebase your change and push again?

**Pull request merged**